web/modules/custom/demo_content/demo_content.install

```diff
@@ -19,6 +20,15 @@ function demo_content_install() {
  * Creates demo content for the joinup project.
  */
 function demo_content_create_demo_content() {
+  // Create demo users.
+  foreach (demo_content_get_user_data() as $user_data) {
+    $facilitator = user_load_by_name($user_data['name']);
```

Comment:
But in order to keep all our demo users together, I am creating them through phing as demo users with only 'authenticated user' role assigned. They are there in every installation of the site.
This module is not overriding anything.